Dropped support for Python 3.8 -- by :user:`Dreamsorcerer`.
